specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/AzureCommunicationServices.json

```diff
@@ -37,6 +37,11 @@
           "description": "Signed incoming call context.",
           "type": "string"
         },
+        "onBehalfOfCallee": {
+          "description": "The communication identifier of the user on behalf of whom the call is made.",
+          "$ref": "../../../../../communication/data-plane/Common/stable/2023-11-15/common.json#/definitions/CommunicationIdentifierModel",
+          "x-ms-client-name": "onBehalfOfCalleeCommunicationIdentifier"
```

Comment:
Why are we renaming this but only in the swagger? I think this name is too long.

Reply:
was trying to make it match other CommunicationIdentifier